graphql/rcos/discord_assoications/account_lookup.graphql

```diff
@@ -0,0 +1,5 @@
+query AccountLookup($user_id: String!) {
+    user_accounts(where: {user_id: {_eq: $user_id}, type: {_eq: "discord"}}, limit: 1) {
+        account_id
+    }
+}
```

Comment:
This file seems very similar to https://github.com/rcos/Telescope/blob/master/graphql/rcos/users/accounts/lookup.graphql, and I think you should use that query instead of this one if possible. Also `user_id`s should be UUIDs instead of Strings.

Reply:
Deleted.